app/workers/annual_gift_removal_worker.rb

```diff
@@ -2,6 +2,9 @@
 
 class AnnualGiftRemovalWorker < ApplicationWorker
   def perform
+    User.elf.with_selected_gifts.pluck(:email, :first_name, :gender).each do |email, first_name, gender|
```

Comment:
I think it would be more clear to pass a full user to the `congratulation_email` method and take necessary fields from it